core/base/src/TStyle.cxx

```diff
@@ -1859,7 +1859,7 @@ void TStyle::SaveSource(const char *filename, Option_t *option)
    char quote = '"';
 
    // Writes include.
-   out << "#if !defined( __CINT__) || defined (__MAKECINT__)" << std::endl << std::endl;
+   out << "#if defined (__MAKECINT__)" << std::endl << std::endl;
```

Comment:
This idiom `!defined( __CINT__) || defined (__MAKECINT__)` meant 'skip' this only for run-time CINT interpretation.  I.e this PR **must** remove the whole thing.
```suggestion
   out << "#if 1" << std::endl << std::endl;
```
(or more exacly remove the `#if` and corresponding `#endif`